sys/src/cmd/mug.c

```diff
@@ -1203,7 +1203,7 @@ main(int argc, char **argv)
 				case 10:
 					state.black += (m.xy.x-lastp.x)/255.0;
 					state.white += (m.xy.x-lastp.x)/255.0;
-					state.gamma = y2gamma(p.y);
+					//state.gamma = y2gamma(p.y); /* Has it sense? Point p never was filled and is not used anymore */
```

Comment:
I think this is wrong. state.gamma is now unitilized. Why not set p to be {0,0} and then use that?